### DISCUSSION TITLE
ways to get caught using this?

### DISCUSSION THREAD

**Discussion by skondar** [nxvvvv/safe-exam-browser-bypass#3] [Q&A] [answered]
Is it possible to get caught using this without someone physically going into your pc/laptop and checking files etc? like monitoring your network or by checking logs that SEB provides etc etc? what are all the ways of getting detected?

**Comment by nxvvvv** [comment, marked as answer]
I mean, you could get caught if you should provide logs but I've provided a solution by editing the logs wherever needed. Just refer the readme.
